infoschema/metric_schema.go

```diff
@@ -925,4 +925,1055 @@ var MetricTableMap = map[string]MetricTableDef{
+		Labels:   []string{"instance"},
+		Quantile: 0.99,
+	},
+	"tikv_lock_manager_deadlock_detect_avg_duration": {
+		PromQL: `sum(rate(tikv_lock_manager_detect_duration_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) / sum(rate(tikv_lock_manager_detect_duration_count{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels: []string{"instance"},
+	},
+	"tikv_lock_manager_detect_error": {
+		PromQL: `sum(rate(tikv_lock_manager_error_counter{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance)`,
+		Labels: []string{"instance", "type"},
+	},
+	"tikv_lock_manager_deadlock_detector_leader": {
+		PromQL: `sum(max_over_time(tikv_lock_manager_detector_leader_heartbeat{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance)`,
+		Labels: []string{"instance"},
+	},
+	"tikv_allocator_stats": {
+		PromQL: `tikv_allocator_stats{$LABEL_CONDITIONS}`,
+		Labels: []string{"instance", "type"},
+	},
+	"tikv_backup_range_size": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_backup_range_size_bytes_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,cf,instance))`,
+		Labels:   []string{"cf", "instance"},
+		Quantile: 0.99,
+	},
+	"tikv_backup_duration": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_backup_request_duration_seconds_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,instance))`,
+		Labels:   []string{"instance"},
+		Quantile: 0.99,
+	},
+	"tikv_backup_avg_duration": {
+		PromQL: `sum(rate(tikv_backup_request_duration_seconds_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) / sum(rate(tikv_backup_request_duration_seconds_count{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels: []string{"instance"},
+	},
+
+	"tikv_backup_flow": {
+		PromQL: `sum(rate(tikv_backup_range_size_bytes_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance)`,
+		Labels: []string{"instance"},
+	},
+	"tikv_disk_read_bytes": {
+		PromQL: `sum(irate(node_disk_read_bytes_total{$LABEL_CONDITIONS}[$RANGE_DURATION]))  by (instance,device)`,
+		Labels: []string{"device", "instance"},
+	},
+	"tikv_disk_write_bytes": {
+		PromQL: `sum(irate(node_disk_written_bytes_total{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance,device)`,
+		Labels: []string{"device", "instance"},
+	},
+	"tikv_backup_range_duration": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_backup_range_duration_seconds_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,type,instance))`,
+		Labels:   []string{"instance", "type"},
+		Quantile: 0.99,
+	},
+	"tikv_backup_range_avg_duration": {
+		PromQL: `sum(rate(tikv_backup_range_duration_seconds_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance) / sum(rate(tikv_backup_range_duration_seconds_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance)`,
+		Labels: []string{"instance", "type"},
+	},
+	"tikv_backup_errors": {
+		PromQL: `rate(tikv_backup_error_counter{$LABEL_CONDITIONS}[$RANGE_DURATION])`,
+		Labels: []string{"error", "instance"},
+	},
```

Comment:
I prefer to move this big map to a new file.

Reply:
great, done.